### ISSUE TITLE
Roadmap for DTS

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#301]
**The problem you're addressing (if any)**

We don't have clear public roadmap for DTS. It is important for various reasons:
- potential contributors can more easily observe what are the plans and align their goals
- it simplifies open-source project management and resource allocation
- it sets priorities for community and contributors

**Describe the solution you'd like**

Please create [milestone](https://github.com/Dasharo/dasharo-issues/milestones) with associated issues, that you plan to address in following and future versions.

Please keep public milestones up to date as well as projects.

**Where is the value to a user, and who might that user be?**

Users will know when to expect given feature.

**Describe alternatives you've considered**

N/A

**Additional context**

**Comment by artur-rs** [comment]
@pietrushnic I believe it's already addressed by milestones: [Dasharo Tools Suite Release v2.0.0](https://github.com/Dasharo/dasharo-issues/milestone/5) and it will be updated based on the DUG#2 presentation.

**Comment by pietrushnic** [comment]
- [ ] Clean up outdated issues
- [ ] Create documentation that clearly explains the DTS development and release process and automate it - this would help get more contributions.
- [ ] Let's switch to publishing artifacts on Github: https://github.com/Dasharo/meta-dts/issues/62
- [ ] Migrate issues from: https://github.com/Dasharo/meta-dts/issues
- [ ] Block creating issues in https://github.com/Dasharo/meta-dts
- [ ] Add in https://github.com/Dasharo/meta-dts/issues README.md that issues should be created in this repo with label DasharoToolsSuite.

**Comment by pietrushnic** [comment]
- [ ] Update milestones deadlines according to: 

![](https://github.com/Dasharo/presentations/raw/dug4/img/dts_roadmap_v0.4.png)

**Comment by artur-rs** [comment]
@pietrushnic 

> Clean up outdated issues

We are in the middle of cleanup. Marked a couple of checkboxes as done

> Create documentation that clearly explains the DTS development and release process and automate it - this would help get more contributions

We should agree on the best place to contribute the process. Right now we have:
1. private repo for dts-release-cicd-pipeline for workflow docs,
2. https://docs.dasharo.com/dasharo-tools-suite/documentation/ where all dev information can be found

I suggest to add new documents/sections to the docs.dasharo instance

> Add in https://github.com/Dasharo/meta-dts/issues README.md that issues should be created in this repo with label DasharoToolsSuite.

Already there: https://github.com/Dasharo/meta-dts?tab=readme-ov-file#reporting-issues
Also issues are migrated and issue-tab closed.

**Comment by pietrushnic** [comment]
> I suggest to add new documents/sections to the docs.dasharo instance

DTS development information should be in meta-dts there is no need to put that under Dasharo. Long term best would be to have Zarhus cover all stuff so nothing specific in DTS regarding release and contribution process would be needed.

> Already there: https://github.com/Dasharo/meta-dts?tab=readme-ov-file#reporting-issues

Great

**Comment by artur-rs** [comment]
@pietrushnic roadmaps and issues are now updated. I've marked the above checkboxes as done.
* v1.2.19 - released, few pings left to close remaining issues
* v1.2.20 - released, closing all finished and moving remaining tasks to v1.2.21
* v1.2.21 - new release scope, consisting of remaining v1.2.20 + part of the v2.0.0
* v2.0.0 - is now an aggregator for the biggest tasks w/o a clear plan. This will be the topic to discuss in the internal planning soon

IMO, the issue can be closed based on this status.
